keps/sig-network/2595-expanded-dns-config/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Expanded DNS Configuration
+kep-number: 2595
+authors:
+  - "@gjkim42"
+owning-sig: sig-network
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: 2021-04-02
+reviewers:
+  - TBD
```

Comment:
@claudiubelu do you know the implications of this resolv.conf changes in windows?